rdf/common/src/main/scala/org/w3/banana/Prefix.scala

```diff
@@ -14,7 +14,7 @@ object Prefix {
     new PrefixBuilder(prefixName, prefixIri)
 }
 
-private [banana] class PrefixBuilder[Rdf <: RDF](
+class PrefixBuilder[Rdf <: RDF](
```

Comment:
I'd like to be able to create Prefixes the way they are done in the file by extending PrefixBuilder

```
class RDFSPrefix[Rdf <: RDF](ops: RDFOps[Rdf]) extends PrefixBuilder("rdfs", "http://www.w3.org/2000/01/rdf-schema#")(ops) {
  val Class = apply("Class")
  val Container = apply("Container")
  ...

```